src/FrontDoor/FrontDoor/Cmdlets/NewFrontDoorWafPolicy.cs

```diff
@@ -90,7 +90,13 @@ public class NewFrontDoorWafPolicy : AzureFrontDoorCmdletBase
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "Custom Response Body")]
         public string CustomBlockResponseBody { get; set; }
-
+        
+        /// <summary>
+        /// The tags to associate with the Front Door Waf Policy.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The tags associate with the Front Door Waf Policy.")]
+        public Hashtable Tag { get; set; }
```

Comment:
Missing using System.Collections; to use hashtable. Please build the proj in you local first to verify the correctness of source code and then submit PR.